docs/2.8.0/docs/kubernetes/03-deploy-dependencies/03-deploy-traefik-ingress-controller.rst

```diff
@@ -43,17 +43,15 @@ Expected output:
      traefik/hub-agent       1.6.0           v1.4.2          Traefik Hub is an all-in-one global networking ...
      traefik/maesh           2.1.2           v1.3.2          Maesh - Simpler Service Mesh
 
-#. Install Traefik:
-
-   The following command expects that you are in the root directory your clone of the `accompanying resources <https://github.com/DACH-NY/daml-enterprise-deployment-blueprints/>`_.
+#. Install Traefik. The following command expects that you are in the root directory of your clone `of the Daml Enterprise Accompanying Resources <https://github.com/DACH-NY/daml-enterprise-deployment-blueprints/>`_:
```

Comment:
I'm ok with changing to something that closely matches what the reader will see when they click the link ("Daml Enterprise Kubernetes Deployment Guide - Accompanying Resources"). A couple challenges are:
- The title they see in the README is very long, but I guess that's okay.
- This link is used in many places throughout the k8s guide. To avoid breaking the docs build, each link description apparently needs to be unique, which requires describing it in many different ways. This leads to a bit of confusion, in my opinion. 

I think changing the link descriptions throughout the guide is workable, but it will take a little time and effort. I'll think about it for a little longer.

Reply:
If you make the links anonymous, then the build should succeed and we can repeat the same anchor text everywhere (note the double underscore at the end):
```code
... in the root directory of your clone `of the Daml Enterprise Deployment Blueprints <https://github.com/DACH-NY/daml-enterprise-deployment-blueprints/>`__:
```

Reply:
thank you...checking into this...

Reply:
I think the anonymous links will be good. Working on a new commit now.